src/libtest/lib.rs

```diff
@@ -76,13 +77,54 @@ mod tests;
 const TEST_WARN_TIMEOUT_S: u64 = 60;
 const QUIET_MODE_MAX_COLUMN: usize = 100; // insert a '\n' after 100 tests in quiet mode
 
+/// This small module contains constants used by `report-time` option.
```

Comment:
The file is pretty large already (2kloc) so it would be good to add the new related changes into a dedicated file.

Reply:
I agree, but I think it will be better to not only move new changes into new file, but refactor the whole file and create several logically-consistent files (e.g. `cli.rs`, `time_options.rs`, `benches.rs`, etc).
I'm going to do that in the next PR. Will it be OK?

Reply:
> I'm going to do that in the next PR. Will it be OK?

Sure, that sounds great!